Assets/HoloToolkit/SpatialMapping/Scripts/SpatialMappingObserver.cs

```diff
@@ -225,21 +328,47 @@ public void CleanupObserver()
 
         /// <summary>
         /// Can be called to override the default origin for the observed volume.  Can only be called while observer has been started.
+        /// Kept for compatibility with Examples/SpatialUnderstanding
         /// </summary>
         public bool SetObserverOrigin(Vector3 origin)
         {
             bool originUpdated = false;
 
             if (observer != null)
             {
-                observer.SetVolumeAsAxisAlignedBox(origin, Extents);
+                Origin = origin;
                 originUpdated = true;
             }
 
             return originUpdated;
         }
 
         /// <summary>
+        /// Change the observed volume according to ObserverVolumeType.
+        /// </summary>
+        private void SwitchObservedVolume()
+        {
+            if (observer != null)
```

Comment:
nit: I'd personally prefer to see an early return here if `observer == null` rather than add a level of indent to the entire function